internal/plans/changes.go

```diff
@@ -210,6 +230,36 @@ func (c *Changes) SyncWrapper() *ChangesSync {
 	}
 }
 
+type QueryInstance struct {
+	Addr addrs.AbsResourceInstance
+
+	ProviderAddr addrs.AbsProviderConfig
+
+	Results cty.Value
```

Comment:
```suggestion
	Results []cty.Value
```

Since this must be a slice / tuple - is it perhaps more clear to just not include that part in the cty encoding? This makes it totally unambiguous that this is a list of things.

Reply:
Currently we're storing the result in the following form: 

```
cty.ObjectVal(map[string]cty.Value{"data": cty.TupleVal(results), "config": config})
```
This matches the schema and makes encoding/decoding easy, but I guess we really only need the tuple val in the plan 🤔